core/src/main/java/org/apache/accumulo/core/iteratorsImpl/system/StatsIterator.java

```diff
@@ -50,27 +52,30 @@ public void next() throws IOException {
     numRead++;
 
     if (numRead % 23 == 0) {
-      readCounter.add(numRead);
+      scanCounter.addAndGet(numRead);
```

Comment:
I was worried about updating two counters on every entry, but the code already does it on every 23 entry so there is nothing to worry about.